phpstan.neon.dist

```diff
@@ -1,22 +1,33 @@
 includes:
-    - php-tools/vendor/szepeviktor/phpstan-wordpress/extension.neon
+    - vendor/szepeviktor/phpstan-wordpress/extension.neon
+
 parameters:
+
     level: 5
+
     inferPrivatePropertyTypeFromConstructor: true
+
+    phpVersion: 70200
+
     dynamicConstantNames:
         - WP_REDIS_SERVERS
+
     bootstrapFiles:
-        - tests/phpstan/bootstrap.php
+        - tests/PHPStan/bootstrap.php
         # Made with generate-stubs --classes --interfaces --traits --out=query-monitor-stubs.php classes/ output/
-        - tests/phpstan/query-monitor-stubs.php
-        - php-tools/vendor/php-stubs/wp-cli-stubs/wp-cli-stubs.php
+        - tests/PHPStan/query-monitor-stubs.php
+        - vendor/php-stubs/wp-cli-stubs/wp-cli-stubs.php
+
+    scanFiles:
+        - php-tools/relay.stub.php
+
     paths:
         - includes/
         - php-tools/object-cache.php
-    excludes_analyse:
+
+    excludePaths:
         - includes/object-cache.php
```

Comment:
Excluding `includes/object-cache.php` seems risky, @szepeviktor?

Reply:
Its the same as `php-tools/object-cache.php`, when we run `analyze` we create a copy of `includes/object-cache.php` do some magic (ref here https://github.com/rhubarbgroup/redis-cache/pull/371#issuecomment-1190724245) and run the analyze on it.